### ISSUE TITLE
Implement touch sensitivity (0xtob/#54)

### ISSUE THREAD

**Issue by asiekierka** [asiekierka/nitrotracker#73]
>Use the touch sensitivity of the DS screen to play notes on the keyboard, or for creating envelopes, or anything else.  The touch sensitivity could be a useful or fun tool for Nitrotracker!

>Yeah, it could be like the note velocity on a MIDI keyboard. Cause then when you record a note to the arrangement it could have its volume set to the note velocity. I doesn't even need to be that good. I could be like an experimental/optional thing you can turn on and off.

**Comment by asiekierka** [comment]
I'm not sure if the touch sensitivity works well enough for usage in a music/keyboard context; further testing is required.

At least we can have it on DSi/3DS now...

**Comment by deadpahn** [comment]
Yeah Im not too crazy about this either. If I need to I use the envelopes or the volume effect.

**Comment by M-O-Marmalade** [comment]
Wait.... the DS can detect the pressure of a touch?!

**Comment by b4ssrp** [comment]
> Wait.... the DS can detect the pressure of a touch?!

Colors! on DS measures touch resistance for sensitivity. 
Actually in libnds there is function called `touchRead()` that returns `z1` and `z2` with description "Raw cross panel resistance."

**Comment by deadpahn** [comment]
Not sure what purpose it would serve honestly. I would vote to add the ability to drag along the keyboard keys to input randomness into the pattern over touch pressure. I used to be able to hit record and move the stylus around to populate notes but no more, just manual tapping. A novice would probably ask WHY but 1337 trackers like myself know that adding variance to hi hats is fun. Gliding the stylus over a couple of keys would be fun and easy to do. Now I have to tap like a woodpecker now on the bus lol
